### PR TITLE
Updated Amalgamation Selection in response to UXA notes

### PR DESCRIPTION
**Pull request by JazzarKarim**
*Issue #:* /bcgov/entity#18805

*Description of changes:*
- Fixes in response to Yui's comment: https://github.com/bcgov/entity/issues/18805#issuecomment-1865014582

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/EntityInfo/EntityMenu.vue [hunk 1]
```diff
@@ -41,34 +41,6 @@
       </v-tooltip>
     </span>
 
-    <!-- Amalgamate -->
-    <span v-if="isBusiness && !isHistorical">
```
Comment (JazzarKarim) [inline src/components/EntityInfo/EntityMenu.vue:45]: Apparently, the design for this was very outdated (before we even implemented the More Actions dropdown).

So, this button here has been moved there and fixed tests accordingly 😁

File: tests/unit/EntityMenu.spec.ts [hunk 2]
```diff
@@ -302,35 +302,27 @@ describe('Entity Menu - Amagamate button tests', () => {
       propsData: { businessId: 'BC1234567' }
     })
 
-    expect(wrapper.find('#amalgamate-button').exists()).toBe(true)
-    expect(wrapper.find('#amalgamate-button').text()).toBe('Amalgamate')
-    expect(wrapper.find('#amalgamate-button').classes()).not.toContain('v-btn--disabled') // enabled
-    wrapper.destroy()
-  })
-
-  it('don\'t display Amalgamate button if business is historical', async () => {
```
Comment (JazzarKarim) [inline tests/unit/EntityMenu.spec.ts:311]: This test is obsolete as the is historical check is now outside (more actions dropdown checks for that).

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-602-o54pzuh0.web.app

File: src/views/AmalgamationSelection.vue [hunk 3]
```diff
@@ -27,7 +27,7 @@
           <v-card
             id="start-horizontal-short-form-card"
             flat
-            class="pt-6 pb-8 px-4"
+            class="pt-8 pb-8 px-8"
```
Comment (severinbeauvais) [inline src/views/AmalgamationSelection.vue:30]: You could combine these 😁 

Also below.

File: tests/unit/EntityMenu.spec.ts [hunk 4]
```diff
@@ -302,35 +302,27 @@ describe('Entity Menu - Amagamate button tests', () => {
...
-    expect(wrapper.find('#amalgamate-button').exists()).toBe(false)
+    expect(wrapper.find('#amalgamate-list-item').exists()).toBe(true)
+    expect(wrapper.find('#amalgamate-list-item').text()).toBe('Amalgamate')
+    expect(wrapper.find('#amalgamate-list-item').classes()).not.toContain('v-list-item--disabled') // enabled
     wrapper.destroy()
   })
 
   it('amalgamate button is disabled if not allowed', async () => {
     businessStore.setLegalType(CorpTypeCd.BC_COMPANY)
-    businessStore.$state.businessInfo.state = EntityState.ACTIVE
```
Comment (severinbeauvais) [inline tests/unit/EntityMenu.spec.ts:324]: It's still good practise to set the variables that you depend on. It's unwise to assume pre-existing data. (I think unit test execution order is not guaranteed.)

**Review by severinbeauvais — APPROVED** [review]
Looks good. OK by me when you've looked at the comments.

**Review by ketaki-deodhar — APPROVED** [review]
👍

**Pull request merged**